crypto/ake/enclave/src/lib.rs

```diff
@@ -709,14 +709,18 @@ impl<EI: EnclaveIdentity> AkeEnclaveState<EI> {
     }
 
     /// Verify attestation evidence
-    pub fn verify_attestation_evidence(
-        &self,
-        attestation_evidence: VerificationReport,
-    ) -> Result<()> {
+    pub fn verify_attestation_evidence(&self, attestation_evidence: EvidenceKind) -> Result<()> {
+        // TODO: This needs to be replaced with dcap verification
```

Comment:
Shouldn't we support both?

Reply:
We only need to support dcap here. But that is out of scope for this PR. This PR is just updating clients to handle both 5.0 and 6.0 networks. This code won't run in production because it will be completely replaced with dcap verification. Assuming Epid here allows builds/tests to succeed.

Reply:
I think this also ties into possibly changing this in-enclave-only API to not use the enum and instead use the actual attestation evidence the enclave supports, since we're never gonna have an enclave supporting both. Does that make sense @dolanbernard @nick-mobilecoin ?

Reply:
I'm on board with that, leaving this VerificationReport and changing it to DcapEvidence when the dcap stuff comes in